aspnet/conceptual-overview/understanding-aspnet5-apps.rst

```diff
@@ -36,192 +29,142 @@ has been refactored into files that are generally smaller and more focused.
 
 .. image:: understanding-aspnet5-apps/_static/references.png
 
+For more information about NuGet, see `Introduction to NuGet <https://docs.asp.net/en/latest/conceptual-overview/nuget.html>`_.
+
 Configuring the Application
 ---------------------------
 
 ASP.NET 5 no longer stores configuration settings in XML files (*web.config* and *machine.config*). Configuration is now stored in *appsettings.json*, which was designed specifically for storing app configuration settings. The default ASP.NET project template includes Entity Framework, and so specifies the database connection string details in the *appsettings.json* file included in the project.
 
-.. code-block:: JSON
-
-	{
-		"Data": {
-			"DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=aspnet5-FundamentalConcepts"
-		},
-
-		"Entity": {
-			"ApplicationDbContext": {
-				"ConnectionString": "Data:DefaultConnection:ConnectionString"
-			}
-		}
-	}
+.. literalinclude:: understanding-aspnet5-apps/sample/src/WebApplication1/appsettings.json
+    :language: json
+    :lines: 2-6,15-19
 
 Individual entries within *appsettings.json* are not limited to name-value pairs, but can specify rich objects. Entries can also reference other entries, as you can see the EF configuration does above.
 
 There's nothing special about the *appsettings.json* filename - it's specified by name in Startup.cs_. You can add as many different configuration files as makes sense for your app, rather than having to add to an ever-growing *web.config* file. You're also not limited to just JSON-formatted files - you can still use XML or even .INI files if you prefer.
 
-Accessing configuration data from your app is best done by injecting the `IConfiguration <https://github.com/aspnet/Configuration/blob/dev/src/Microsoft.Framework.ConfigurationModel.Interfaces/IConfiguration.cs>`_ interface into your controller, and then simply using the indexer with the name of the configuration element you need. For example, to store the application name in config and display it on the About page, you would need to make three changes to the default project. First, add the entry to *appsettings.json*.
-
-.. code-block:: JSON
-	:emphasize-lines: 11
-
-	{
-		"Data": {
-			"DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=aspnet5-FundamentalConcepts"
-		},
+Accessing configuration data from your app is best done by injecting the `IConfiguration <https://github.com/aspnet/Configuration/blob/dev/src/Microsoft.Extensions.Configuration.Abstractions/IConfiguration.cs>`_ interface into your controller, and then simply using the indexer with the name of the configuration element you need. For example, to store the application name in config and display it on the **About** page, you would need to make three changes to the default project.
```

Comment:
Injecting IConfiguration actually isn't best practice - folks should use the options model instead.

Better examples of DI that already exists in the default templates would be some of the services used by ASP.NET Identity, like `IEmailSender` and `ISmsSender`. These are injected into the AccountsController along with with other Identity services and the `DBContext` used for account management.